src/holodeck/environments.py

```diff
@@ -255,6 +255,21 @@ def _load_scenario(self):
             self.agents[agent['agent_name']].set_control_scheme(agent['control_scheme'])
             self._spawned_agent_defs.append(agent_def)
 
+        if "weather" in self._scenario:
+            weather = self._scenario["weather"]
+            if "hour" in weather:
+                self.weather.set_day_time(weather["hour"])
+            if "type" in weather:
+                self.weather.set_weather(weather["type"])
+            if "fog_density" in weather:
+                self.weather.set_fog_density(weather["fog_density"])
+            if "day_cycle_length" in weather:
+                day_cycle_length = weather["day_cycle_length"]
+                if day_cycle_length <= 0:
```

Comment:
So if `day_cycle_length` is less than 0 it will stop the day cycle? This is a little strange but I think it could be useful if we ever add a world that has a day cycle by default. It should be documented in the weather doc though.

Reply:
Would it be better if we only stopped the day cycle if the cycle length is exactly equal to 0? Looking at this again I'm not really sure why we wouldn't just throw an error if someone passed in a negative value.

Reply:
Good catch on documentation though. I'll make sure we have something for that on the new weather usage page.